src/server.c

```diff
@@ -256,6 +258,24 @@ server_init(struct server *server)
 		exit(EXIT_FAILURE);
 	}
 
+	/*
+	 * Create headless backend to enable adding virtual outputs later on.
+	 */
+	server->headless.backend = wlr_headless_backend_create(server->wl_display);
+	if (!server->headless.backend) {
+		wlr_log(WLR_ERROR, "failed to create virtual output");
+		exit(EXIT_FAILURE);
+	}
+	wlr_multi_backend_add(server->backend, server->headless.backend);
+
+	/*
```

Comment:
Sway creates a "FALLBACK" headless output at this point but does not immediate destroy it.
Do we understand why? Are your requirements different?

Reply:
AFAIR the fallback virtual output in sway is a hack so they don't have to deal with a situation where they don't have any more outputs around. Last I heard about that was that they want to remove it.